doc/modules/preprocessing.rst

```diff
@@ -294,13 +299,17 @@ centering :math:`K_{test}` is done as:
 `(n_samples_test, n_samples)` where all entries are equal to
 :math:`\frac{1}{\text{n}_{samples}}`.
 
+|details-start|
 .. topic:: References
+|details-split|
```

Comment:
The reference [Scholkopf1998] is being called from the `Mathematical formulation` dropdown. As this is the only part of the document that refers to it, it's ok as long as you keep both the call and the reference inside the same dropdown, i.e.

```suggestion
.. topic:: References
```